lib/plugins/aws/provider/awsProvider.js

```diff
@@ -504,6 +504,53 @@ class AwsProvider {
           },
           additionalProperties: false,
         },
+        layers: {
+          type: 'object',
+          additionalProperties: {
+            type: 'object',
+            properties: {
+              allowedAccounts: {
+                type: 'array',
+                items: {
+                  oneOf: [
+                    { type: 'integer', minimum: 100000000000, maximum: 999999999999 },
```

Comment:
Today I've turned on automatic coercion with https://github.com/serverless/serverless/pull/8319, and having that I believe we  no longer need that `number` definition.

We can also remove the logic which tries to handle eventual numeric input